internal/expand/handler.go

```diff
@@ -76,6 +77,10 @@ type getExpandRequest struct {
 //       404: genericError
 //       500: genericError
 func (h *handler) getExpand(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	if !r.URL.Query().Has("max-depth") {
+		h.d.Writer().WriteError(w, r, herodot.ErrBadRequest.WithError(`required parameter "max-depth" is missing`))
```

Comment:
I suggest you adjust the error message here:
```suggestion
		h.d.Writer().WriteError(w, r, herodot.ErrBadRequest.WithError("required query parameter 'max-depth' is missing"))
```
which will make it a bit more helpful and also avoid double-escaping in the test.